.travis.yml

```diff
@@ -1,28 +1,40 @@
 language: ruby
 rvm:
-- 2.2.5
-- 2.5.1
+- 2.6
+- 2.5
+- 2.4
+# We test up to the latest version of Ruby here, and test these Ruby versions
+# against all support ActiveRecord versions. One test will be "duplicated"
```

Comment:
Let me know if you have a better solution here!

Reply:
Meh, makes sense to me!